includes/module/model/TicketCheckout.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
```

Comment:
Colocar NOTICE OF LICENSE no header do arquivo (VIDE mercadopago.php)

Reply:
Toppen